wormchain/contracts/tools/deploy_wormchain.ts

```diff
@@ -0,0 +1,333 @@
+import "dotenv/config";
+import { SigningCosmWasmClient, toBinary, InstantiateResult } from "@cosmjs/cosmwasm-stargate";
+import { GasPrice } from "@cosmjs/stargate"
+import { Secp256k1HdWallet } from "@cosmjs/amino";
+import { MsgExecuteContract } from "cosmjs-types/cosmwasm/wasm/v1/tx";
+import * as fs from "fs";
+import { readdirSync, } from "fs";
+import * as util from 'util'
+import { Bech32, toHex, toUtf8, fromBase64 } from "@cosmjs/encoding";
+import { zeroPad, keccak256 } from "ethers/lib/utils.js";
+import * as elliptic from "elliptic"
+
+import * as devnetConsts from "./devnet-consts.json"
+import { concatArrays, encodeUint8 } from "./utils";
+
+if (process.env.INIT_SIGNERS === "undefined") {
+  let msg = `.env is missing. run "make contracts-tools-deps" to fetch.`
+  console.error(msg)
+  throw msg
+}
+
+const readFileAsync = util.promisify(fs.readFile);
+
+/*
+  NOTE: Only append to this array: keeping the ordering is crucial, as the
+  contracts must be imported in a deterministic order so their addresses remain
+  deterministic.
+*/
+type ContractName = string
+const artifacts: ContractName[] = [
+  "wormhole.wasm",
+  "token_bridge_terra_2.wasm",
+  "cw20_wrapped_2.wasm",
+  "cw20_base.wasm",
+  "mock_bridge_integration_2.wasm",
```

Comment:
Now that #2121 is merged, we don't actually need to deploy any of these.  We only need to deploy the accounting contract itself.

Reply:
I decided to leave the core contract and token bridge deploy since it is already written, and we'll likely want it in devnet in the future.

Reply:
I don't think there is any situation where we will want to deploy the core contract on wormchain because wormchain has the native core module, which does everything the core contract does.

Reply:
+1, no need to deploy any of the bridge contracts for now. I think the only one we need on wormchain is the accounting contract.

Reply:
Sure, I'll update the deploy script to only deploy the accountant contract.

Reply:
done, resolving